RoaringBitmap/src/test/java/org/roaringbitmap/longlong/TestRoaring64Bitmap.java

```diff
@@ -505,6 +505,15 @@ public void testSerializationEmpty() throws IOException, ClassNotFoundException
     assertEquals(0, clone.getLongCardinality());
   }
 
+  @Test
+  public void testSerialization_ToBigEndianBuffer()throws IOException, ClassNotFoundException {
```

Comment:
I think this test case should have a code format. Please run `./gradlew checkstyleMain` first.

Reply:
I have run the check style, no issue was highlighted. but I figured it needed space after function declaration. may somebody has to check the validation rules.

Reply:
The line 512, 514 also needs a space too.

Reply:
@weijietong  please check now. sorry for the trouble, for my Netbeans 11  IDE,  I couldn't find a plugin that supports google style.